### DISCUSSION TITLE
WHO ELSE IS UNABLE TO INSTALL CHAINLINK BROWNIE CONTRACT VERSION 1.1.1  IN 2024?

### DISCUSSION THREAD

**Discussion by Saber1Y** [Cyfrin/foundry-full-course-cu#2231] [Q&A] [answered]
![Code_Lcmhf5EPi0](https://github.com/user-attachments/assets/9c5ea711-02af-4e45-bd91-a6b5f62f7a54)

ERROR: 
forge install smartcontractkit/chainlink-brownie-contracts@1.1.1 --no-commit
Installing chainlink-brownie-contracts in /home/saberthedev/foundry_lottery/lib/chainlink-brownie-contracts (url: Some("https://github.com/smartcontractkit/chainlink-brownie-contracts"), tag: Some("1.1.1"))
Cloning into '/home/saberthedev/foundry_lottery/lib/chainlink-brownie-contracts'...
remote: Enumerating objects: 34274, done.
remote: Counting objects: 100% (3875/3875), done.
remote: Compressing objects: 100% (1140/1140), done.
error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8)
error: 5465 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/smartcontractkit/chainlink-brownie-contracts' into submodule path '/home/saberthedev/foundry_lottery/lib/chainlink-brownie-contracts' failed
Error:

**Comment by EngrPips** [comment]
Hello @Saber1Y, This is an internet connection problem. Please retry when the internet connection is stable.

**Reply by Saber1Y** [reply]
it's been 3 days and it's not still installing, how slow can my internet be..sigh,,,

**Reply by EngrPips** [reply, marked as answer]
It might not be very slow, but it also needs to be very stable throughout the process, so you should try when your internet connection is fast and stable.

**Comment by Saber1Y** [comment]
Yea i think i need stable internet speed, thanks!

**Reply by EngrPips** [reply]
You are welcome, Friend.
